test/integration/dialects/postgres/dao.test.js

```diff
@@ -965,7 +965,7 @@ if (dialect.startsWith('postgres')) {
     it('should save geometry correctly', async function () {
       const point = { type: 'Point', coordinates: [39.807_222, -76.984_722] };
       const newUser = await this.User.create({ username: 'user', email: ['foo@bar.com'], location: point });
-      expect(newUser.location).to.deep.eql(point);
+      expect(newUser.location).to.deep.include(point);
```

Comment:
Why this change?

Reply:
Postgis was updated to V3 which includes more metadata, but V2 is still supported, which does not include it